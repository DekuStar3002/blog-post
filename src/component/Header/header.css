.header {
  background-color: black;
  color: whitesmoke;
}

.logo {
  display: flex;
  align-items: flex-start;
  justify-content: flex-start;
  flex-direction: column;
  padding: 2vh 5vw;
}

.logo :nth-child(1) {
  font-size: x-large;
}

.logo :nth-child(2) {
  font-style: italic;
}

.navbar {
  display: flex;
  align-items: flex-end;
  justify-content: flex-end;
}

.navbar a {
  text-decoration: none !important;
  color: whitesmoke;
  margin: 3vh 5vw;
}

a {
  text-decoration: none;
}