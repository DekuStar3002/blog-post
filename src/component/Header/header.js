import React from 'react';
import './header.css';
const Header = () => {
  return (
    <div className='header'>
      <div className='logo'>
        <p>The Artifact</p>
        <p>Culture & Art blog</p>
      </div>
      <div className='navbar'>
        <a href='#blog'>Blog</a>
        <a href='#about'>About</a>
        <a href='#contact'>Contact</a>
      </div>
    </div>
  );
}

export default Header;