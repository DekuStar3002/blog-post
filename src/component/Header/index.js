import React from 'react';
import './Header.css';
const Header = () => {
  return (
    <div className='header'>
      <div className='logo'>
        <p>The Artifact</p>
        <p>Culture & Art blog</p>
      </div>
      <div className='navbar'>
        <a href='#blog'>Blog</a>
        <a href='#about'>About</a>
        <a href='#contact'>Contact</a>
      </div>
    </div>
  );
}

export default Header;