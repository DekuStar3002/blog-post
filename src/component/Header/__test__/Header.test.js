import Header from '..';
import { render } from '@testing-library/react';

describe('Header', () => { 
  it('should render correctly', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  })
});