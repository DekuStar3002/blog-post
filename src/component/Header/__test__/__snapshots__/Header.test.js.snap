// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <div
      class="logo"
    >
      <p>
        The Artifact
      </p>
      <p>
        Culture & Art blog
      </p>
    </div>
    <div
      class="navbar"
    >
      <a
        href="#blog"
      >
        Blog
      </a>
      <a
        href="#about"
      >
        About
      </a>
      <a
        href="#contact"
      >
        Contact
      </a>
    </div>
  </div>
</DocumentFragment>
`;
